test/unit/location_test.rb

```diff
@@ -231,4 +231,8 @@ def province_code
     assert_equal @location, another_location
     refute_equal @location, Location.new({})
   end
+
+  test "#== works with nil" do
+    refute_equal @location, nil
```

Comment:
`refute_nil` could be used here

Reply:
My bad, using `refute_nil` is not ok, as we don't want to check that the value is not `nil` (which I think should not happen), but that the comparison with `nil` returns `false` 😅